seqr/views/apis/report_api_tests.py

```diff
@@ -509,10 +509,7 @@
     'notes': None,
     'phenotype_contribution': 'Full',
     'partial_contribution_explained': '',
-    'phenotype_description': None,
-    'pmid_id': None,
     'seqr_chosen_consequence': None,
-    'solve_status': 'Unsolved',
```

Comment:
these fields are not used on the variant metadata page even though they are included in the response, at some point I will want to go back and remove everything we are not using from the response to prevent confusion in cases like this one but thats out of scope here